src/libcore/ops/function.rs

```diff
@@ -10,24 +10,37 @@
 
 /// A version of the call operator that takes an immutable receiver.
 ///
-/// # Examples
+/// Closures only taking immutable references to captured variables
+/// automatically implement this trait, which allows them to be invoked.
```

Comment:
the "which allows them to be invoked" feels awkward to me, but I'm not sure I have good suggestions.